sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample08_Administration.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Identity;
+using NUnit.Framework;
+using Azure.Messaging.ServiceBus.Administration;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample08_Administration : ServiceBusLiveTestBase
```

Comment:
Let's add this to Sample07_CrudOperations or we can even rename it to be Sample07_Administration if you'd like.

Reply:
I have been dropping into this in 30s chunks through the day, and I must have had momentary madness each time. I won't rename the file (but I think it compounded my confusion), I don't want to break any potential incoming links.